_extensions/posit-docs/theme-dark.scss

```diff
@@ -93,6 +98,56 @@ $body-bg: $dark-md-background !default; // dark mode change
   background-color: #0f1c29 !important;
 }
 
+/* Search results */
+
+.aa-DetachedOverlay .aa-SourceHeader .search-result-header, #quarto-search-results .aa-SourceHeader .search-result-header {
```

Comment:
I am surprised that we need to style such specific things. I would have expected Quarto to make this type of adjustment smoother through variables.

Reply:
I know that we just had a quick slack chat but yeah - the colors are inherited by variables that we have set. Our dark theme `$primary` color is `$primary: $posit-teal;` and the "background" of the search was showing that teal...which was terrible.

So the colors that search is using are all set with variables however, there isn't a clean way to override those things without having to dig into this gross CSS to do so.

I looked through bootstrap and I didn't see any type of search results/box customization nor did I in Quarto :(